cmd/admin/admin.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"github.com/dubbogo/dubbo-go-proxy/pkg/common/yaml"
```

Comment:
please split the import by the package source. standard packages, external packages and internal packages.

Reply:
ok